index.js

```diff
@@ -151,20 +151,45 @@ var Zetkin = function() {
 
     /**
      * Retrieve a resource proxy through which requests to that resource can be
-     * made.
+     * made. An optional query object may be added as the last argument.
      *
      * Example: Z.resource('orgs', 1, 'people').get() will make a HTTP GET
      * request to the /orgs/1/people resource.
+     *
+     * Example: Z.resource('orgs', 1', 'sub_organizations', { recursive: true }) u
+     * will make a HTTP GET request to /orgs/1/people?recursive
     */
     this.resource = function() {
-        path = Array.prototype.join.call(arguments, '/');
+        args = Array.prototype.filter.call(arguments, arg => typeof(arg) === 'string' || typeof(arg) === 'number');
+        path = args.join('/');
         if (path.length == 0 || path[0] != '/') {
             path = '/' + path;
         }
 
         path = _config.base + '/v' + _config.version + path;
 
-        return new ZetkinResourceProxy(this, path, _request);
+        query = [];
+
+        if(args.length == arguments.length-1) {
+            queryObject = arguments[arguments.length-1]
+            if(typeof(queryObject) !== 'object') {
+                throw new Error('Only the last argument of resource may be an object!')
+            }
+            query = Object.keys(queryObject)
+                // Filter any keys with values that evaluate to false
+                .filter(key => !!queryObject[key])
+                .map(key => {
+                    if(queryObject[key] === true) {
+                        return key;
+                    } else {
+                        return key + '=' + queryObject[key];
```

Comment:
How does it deal with escaping/URL encoding? What happens if you pass an object like `{ foo: "bar&baz" }`?

Reply:
Yup, this could present problems, I will add url encoding.